nomos-core/chain-defs/src/wire_errors.rs

```diff
@@ -0,0 +1,16 @@
+//STD
+use std::io;
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Failed to serialize message: {0}")]
+    Serialize(bincode::Error),
+    #[error("Failed to deserialize message: {0}")]
+    Deserialize(bincode::Error),
+}
```

Comment:
Usually unless it is too convoluted this could live in wire. why? it is usually easier to import everything from the same module:
```rust
use wire::{serialize, deserialize, Error};
```